trpcultivate_phenotypes/trpcultivate_phenotypes.services.yml

```
@@ -14,4 +14,4 @@ services:
 
   trpcultivate_phenotypes.traits:
     class: 'Drupal\trpcultivate_phenotypes\Service\TripalCultivatePhenotypesTraitsService'
-    arguments: ['@trpcultivate_phenotypes.genus_ontology', '@trpcultivate_phenotypes.terms', '@tripal_chado.database', '@tripal.logger']
+    arguments: ['@trpcultivate_phenotypes.genus_ontology', '@trpcultivate_phenotypes.terms', '@tripal_chado.database', '@tripal_chado.chado_buddy', ]
```

```suggestion
    arguments: ['@trpcultivate_phenotypes.genus_ontology', '@trpcultivate_phenotypes.terms', '@tripal_chado.database', '@tripal_chado.chado_buddy']
```
The final comma symbol can be safely left out.